doc/index.md

```diff
@@ -13,8 +13,8 @@
 ```bash
 $ composer require amenophis/elasticsearch ^1.0@dev
 ```
-### Configuring in a Symfony Flex application
-You must to register the bundle in the `config/bundles.php`:
+## Configuration
+Register the bundle in the `config/bundles.php`:
```

Comment:
FYI bundle is registered automatically with Flex, even if you don't have any recipe.